src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.NETFramework.targets

```diff
@@ -83,13 +74,14 @@ Copyright (c) .NET Foundation. All rights reserved.
            Simple references can also come from NuGet framework assemblies, hence this statement should occur after
            including all computed references, thus this target is scheduled after references have been raised by NuGet
            targets. -->
-      <Reference Remove="%(_NETStandardLibraryNETFrameworkReference.FileName)" />
+      <Reference Remove="%(_NETStandardLibraryNETFrameworkLib.FileName)" />
 
-      <Reference Include="@(_NETStandardLibraryNETFrameworkReference)">
+      <Reference Include="@(_NETStandardLibraryNETFrameworkLib.FileName)">
+        <HintPath>%(_NETStandardLibraryNETFrameworkLib.Identity)</HintPath>
         <Private>false</Private>
       </Reference>
-
-      <ReferenceCopyLocalPaths Include="@(_NETStandardLibraryNETFrameworkLib)">
+      <!-- netfx.force.conflicts is only needed at compile time. -->
+      <ReferenceCopyLocalPaths Include="@(_NETStandardLibraryNETFrameworkLib)" Condition="'%(FileName)' != 'netfx.force.conflicts'">
```

Comment:
Can this be conditioned differently? Why didn't we need to know about netfx.force.conflicts here before, but now we do?

Reply:
I see now. This is just porting what was there before revert to new scheme.